src/Helpers.py

```diff
@@ -143,15 +154,49 @@ def get_items_with_value(world: World, multiworld: MultiWorld, value: str, playe
 
     value = value.lower().strip()
 
-    if not hasattr(world, 'item_values'): #Cache of just the item values
-        world.item_values = {}
+    if not skipCache:
+        if not hasattr(world, 'item_values'): #Cache of just the item values
+            world.item_values = {}
 
-    if not world.item_values.get(player):
-        world.item_values[player] = {}
+        if not world.item_values.get(player):
+            world.item_values[player] = {}
 
-    if value not in world.item_values.get(player, {}).keys() or force:
+    if value not in world.item_values.get(player, {}).keys() or skipCache:
         item_with_values = {i.name: world.item_name_to_item[i.name]['value'].get(value, 0)
                             for i in player_items if i.code is not None
                             and i.name in world.item_name_groups.get(f'has_{value}_value', [])}
+        if skipCache:
+            return item_with_values
         world.item_values[player][value] = item_with_values
     return world.item_values[player].get(value)
+
+def filter_used_regions_for_player(world: World, player_regions: dict|list, player: Optional[int] = None, ) -> set:

```

Comment:
Trailing comma

Reply:
That's from when I moved player_regions from optional to instead be required.(looping the multiworld regions in that function somehow created a potential memory leak warning from ap)
I'll fix that when I'm back to my PC.

Reply:
neither player or world are actually used now that the multiworld.regions stuff is done outside of it so I removed them at the same time I fixed this typo